torch/_torch_docs.py

```diff
@@ -3559,6 +3559,22 @@ def merge_dicts(*dicts):
 
     >>> torch.normal(mean=torch.arange(1., 6.))
     tensor([ 1.1552,  2.6148,  2.6535,  5.8318,  4.2361])
+
+.. function:: normal(mean, std, size, *, out=None) -> Tensor
+
+Similar to the function above, but the means and standard deviations are shared
+among all drawn elements. The resulting tensor has size given by :attr:`size`.
+
+Args:
+    mean (float): the mean for all distributions
+    std (float): the standard deviation for all distributions
+    size (int...): a sequence of integers defining the shape of the output tensor.
```

Comment:
can we add size everywhere?  That seems consistent and doesn't need yet another overload.

Reply:
We would need the extra overload anyway, to support all of `normal(Tensor, float, sizes)`, `normal(float, Tensor, sizes)`, and `normal(float, float, sizes)`

Reply:
can't size have a default parameter?  So you would have only the overloads that exist now.

Reply:
My point is we don't have any overload now that takes two floats. We have overloads that take two tensors, or one tensor and one float.

Reply:
sorry I missed that.  I still think we should just add size everywhere, though.

Reply:
For posterity: gchanan and I spoke about this offline. We agreed that we do need to have a new override for the "two floats" case, but there's also no good reason not to add "sizes" everywhere. So I'll do that.